project/Mima.scala

```diff
@@ -6,13 +6,14 @@ import com.typesafe.tools.mima.core._
 // More details about Mima:
 // https://github.com/typesafehub/migration-manager/wiki/sbt-plugin#basic-usage
 object Mima {
-  val ignoredABIProblems: Seq[ProblemFilter] = {
-    Seq(
```

Comment:
How do we manually exclude/ignore individual problems? I think we should still keep this `Seq` so we can copy-paste the mima errors when we intentionally break the API. Example when we need to manually exclude errors, package private members are breaking changes on the bytecode level but according to the language contract it's not a breaking change.

Reply:
Let's do this on by-need basis. Now that we have a compatibility policy, adding these ignores won't be trivial anyway, since we'll likely need to update the policy, too.

Reply:
changes to package private members are within our compatibility policy and they will require custom ProblemFilters, but sure we can deal with it then